src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/TabularEnergyMultipliers/Builder/test/Builder.test.cpp

```diff
@@ -0,0 +1,101 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "ACEtk.hpp"
+
+using namespace njoy::ACEtk;
+using namespace njoy::ACEtk::interpretation;
+
+SCENARIO( "Testing EnergyDistribtion::TabularLinearMultipliers::Builder" ){
+    ContinuousEnergyNeutron::Builder CENBuilder;
+
+    auto parentBuilder = CENBuilder.reaction( 14 ).energyDistribution();
+    using LAW24Builder = decltype( parentBuilder.tabularEnergyMultipliers() );
+
+    struct TestBuilder : public LAW24Builder {
+      using LAW24Builder::construct;
+      using LAW24Builder::LAW24Builder;
+    };
+
+
+  std::vector< int > boundaries{ 0, 3 };
+  std::vector< int > schemes{ 2, 1 };
+  std::vector< double > energies{ 1.0, 2.0, 5.0, 6.0 };
+  std::vector< double > T1{ 1.1, 1.2, 1.3 };
+  std::array< double, 3 > T2{{ 1.1, 1.2, 1.4 }};
+
+  TestBuilder law24( parentBuilder );
+
+  GIVEN( "valid inputs" ){
+
+    law24.boundaries( njoy::utility::copy( boundaries ) )
+         .schemes( njoy::utility::copy( schemes ) )
+         .energies( njoy::utility::copy( energies ) )
+         .table( njoy::utility::copy( T1 ) )
+         .table( njoy::utility::copy( T1 ) )
+         .table( njoy::utility::copy( T2 ) )
+         .table( njoy::utility::copy( T2 ) );
+
+      auto distribution = law24.construct();
+      THEN( "the values can be verified" ){
+        auto parameters = distribution.parameters;
+        CHECK( ranges::equal( boundaries, parameters->first ) );
+        CHECK( ranges::equal( schemes, parameters->second ) );
+        CHECK( ranges::equal( energies, distribution.x ) );
+
+        auto y = distribution.y;
+        CHECK( ranges::equal( T1, y[0] ) );
+        CHECK( ranges::equal( T1, y[1] ) );
+        CHECK( ranges::equal( T2, y[2] ) );
+        CHECK( ranges::equal( T2, y[3] ) );
+      }
+
```

Comment:
There is no ACEify test, and I think that NET will not be printed, as with LAW=22.

Reply:
It does now.